src/main/java/org/sobotics/guttenberg/clients/Guttenberg.java

```diff
@@ -182,4 +198,49 @@ private void checkLastExecution() {
 		}
 		
 	}
+	
+	/**
+	 * Checks for updates
+	 * */
+	private void update() {
+		System.out.println("Load updater...");
+		Updater updater = new Updater();
+		System.out.println("Check for updates...");
+		
+		int update = updater.updateIfAvailable(); 
+		
+		if (update == -1) {
+			for (Room room : this.chatRooms) {
+				if (room.getRoomId() == 111347) {
+					room.send("Automatic update failed!");
+				}
+			}
+		} else if (update == 1) {
+			for (Room room : this.chatRooms) {
+				if (room.getRoomId() == 111347) {
+					room.send("Rebooting for update to version "+updater.getNewVersion().get());
+				}
+				room.leave();
+			}
+			System.exit(0);
+		}
```

Comment:
Why not refactor this so that there is only one `for` loop, and the `if` inside?